### DISCUSSION TITLE
It is possible compiles HiGHS to web assembly?

### DISCUSSION THREAD

**Discussion by sebastianherreramonterrosa** [ERGO-Code/HiGHS#2046] [Q&A]
I am interested in running HiGHS in the browser with python interface. I would like to know if something like this is possible and if so, what would be the way to achieve it. In fact, any tool on the subject would be useful.

**Comment by jajhall** [comment]
It can be run within Colab or Jupyter notebooks, if that's what you mean.

If you mean ordinary web pages, there's a JavaScript interface.

Let me know more and I can advise you further

**Comment by odow** [comment]
See https://github.com/lovasoa/highs-js

**Comment by airen1986** [comment]
You can build highs wheel following instructions at: https://pyodide.org/en/stable/development/building-and-testing-packages.html

I have built it successfully (1.8.0). You can download it from: https://static.supplychainlite.com/wheels/highspy-1.8.0-cp312-cp312-pyodide_2024_0_wasm32.whl
